### PR TITLE
Create replit_token_storage.py

### PR DESCRIPTION
**Pull request by alexyy802**
Created replit token storage for repl.it users to make their token a secret

## Summary

replit token storage for repl.it users to make their token a secret

## Checklist

- [x] If code changes were made then they have been tested.
    - [x ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [x] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: examples/replit_token_storage.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+@bot.event
+async def on_ready():
+  print('Bot online!')
+ 
+@bot.command()
+async def ping(ctx):
+  await ctx.send('Ping!')
+
+# In repl.it there is a secret variable witch you can store ur secrets
```
Comment (VincentRPS) [inline examples/replit_token_storage.py:15]:
```suggestion
# In repl.it there is a secret variable witch you can store your secrets
```

File: examples/replit_token_storage.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# You need to navigate to a lock icon on the left side of ur screen
```
Comment (VincentRPS) [inline examples/replit_token_storage.py:16]:
```suggestion
# You need to navigate to a lock icon on the left side of your screen
```

File: examples/replit_token_storage.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# After that you schould see a field called "key" thats where you put the name of ur token for example "token" or "discord-bot-token"
```
Comment (VincentRPS) [inline examples/replit_token_storage.py:17]:
```suggestion
# After that you should see a field called "key" that's where you your token for example "token" or "discord-bot-token"
```

File: examples/replit_token_storage.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+# On the bottom of the key you schould see another field called "value" that is where you put ur bot token
```
Comment (VincentRPS) [inline examples/replit_token_storage.py:18]:
```suggestion
# On the bottom of the key you should see another field called "value" that is where you put your bot token
```

File: examples/replit_token_storage.py [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+# After that press the Add new secret button and ur done!
```
Comment (VincentRPS) [inline examples/replit_token_storage.py:19]:
```suggestion
# After that press the Add new secret button and your done!
```

**Review by VincentRPS — CHANGES_REQUESTED** [review]
Multiple Spelling Mistakes And Didn't https://github.com/nextcord/nextcord/commit/6b2254c2e9beef48303213903fe72a6637b1c7f9 already add this?

File: examples/replit_token_storage.py [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  
+token = os.environ['token']
```
Comment (Skelmis) [inline examples/replit_token_storage.py:21]: This should be ``token = os.environ["TOKEN"]``

File: examples/replit_token_storage.py [hunk 7]
```diff
@@ -0,0 +1,22 @@
+import nextcord
+from nextcord.ext import commands
+import os
```
Comment (Skelmis) [inline examples/replit_token_storage.py:3]: ``import os`` should be the first line, followed by a line gap then the nextcord imports inline with pep8

**Review by Skelmis — CHANGES_REQUESTED** [review]
I do not see the point in this?
If anything it could be a note in the beginner documentation rather then an example given it pertains to environ

File: examples/replit_token_storage.py (hunk 6 above)
Comment (ooliver1) [reply examples/replit_token_storage.py:21]: The rest of the lib is in single quotes as of rn so no vincent

Comment (VincentRPS) [reply examples/replit_token_storage.py:21]: @ooliver1 #232 should be changing that

**Comment by ooliver1** [comment]
I don't see much point, as replit users are using replit most likely from a tutorial and they usually show you the secrets or a .env file

File: examples/replit_token_storage.py (hunk 3 above)
Comment (ooliver1) [reply examples/replit_token_storage.py:17]: Where you your token

File: examples/replit_token_storage.py (hunk 1 above)
Comment (alexyy802) [reply examples/replit_token_storage.py:15]: Yea but not for repl.it

**Comment by alexyy802** [comment]
> I don't see much point, as replit users are using replit most likely from a tutorial and they usually show you the secrets or a .env file

True

**Comment by TAG-Epic** [comment]
We already have a secret storage example.

**Pull request closed without merging**